code/game/objects/buckling.dm

```diff
@@ -5,6 +5,7 @@
 	var/buckle_lying = -1 //bed-like behavior, forces mob.lying = buckle_lying if != -1
 	var/buckle_require_restraints = 0 //require people to be handcuffed before being able to buckle. eg: pipes
 	var/mob/living/buckled_mob = null
+	var/has_buckled

```

Comment:
To remove any possible fuckery where a mob is removed from the buckled_mob but is still technically buckled. If that was the case, then my added var wouldn't have been changed, and would still recognise there's a mob buckled, which there is.